Packs/CoreAlertFields/IncidentFields/incidentfield-Host_FQDN.json

```diff
@@ -28,5 +28,19 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
```

Comment:
Need to validate if this can be multiselect

Reply:
@David-BMS - pls update once you have an answer